.github/workflows/ci.yaml

```diff
@@ -73,7 +73,7 @@ jobs:
       - name: GolangCI Lint
         uses: golangci/golangci-lint-action@v3
         with:
-          version: v1.49.0
+          version: v1.53.0
```

Comment:
This build still shows the same errors and warnings:
https://github.com/RamenDR/ramen/actions/runs/5609830827

What do you mean by "Publish Built Image warnings"? Maybe share links before/after this change that will make it more clear?

Reply:
I was referring to the warnings here: https://github.com/RamenDR/ramen/actions/runs/5607722950